pandas/core/generic.py

```diff
@@ -5851,6 +5853,10 @@ def sample(
         -----
         If `frac` > 1, `replacement` should be set to `True`.
 
+        When replace = False will not allow ``(n * max(weights) / sum(weights)) > 1``,
+        since that would cause results to be biased. E.g. sampling 2 items without replacement,
+        with weights [100, 1, 1] would yield two last items in 1/2 of cases, instead of 1/102
```

Comment:
```suggestion
        with weights ``[100, 1, 1]`` would yield two last items in 1/2 of cases, instead of 1/102.
```

Reply:
Can you also add, "This is similar to specifying ``n=4`` without replacement on a Series with 3 elements."